source/manual/rabbitmq.html.md

```diff
@@ -4,147 +4,123 @@ title: Manage RabbitMQ
-As root run:
+[RabbitMQ][RabbitMQ] is a message broker based on the [Advanced Message Queuing
+Protocol][AMQP] (AMQP).
 
-```bash
-$ rabbitmqctl eval 'rabbit_federation_status:status().'
-```
-
-On one of the nodes, you should get: `{status,running}`
-
-If not something is wrong and the federation is broken.
-Check the logs in /var/log/rabbitmq and verify that the credentials and
-IPs address for the federation are correct by running as root :
-
-```bash
-$ rabbitmqctl list_parameters
-```
-
-## Viewing RabbitMQ metrics
-
-Metrics from RabbitMQ are collected with a CollectD plugin, and are
-available in Graphite/Grafana. There is a [generic RabbitMQ
-dashboard][rabbitmq-dashboard] which shows the main metrics for queues
-and exchanges.
-
-[rabbitmq-dashboard]: https://grafana.publishing.service.gov.uk/dashboard/file/rabbitmq.json
+[Learn more about RabbitMQ][rabbitmq_tutorial].
 
 ## Connecting to the RabbitMQ web control panel
 
-1. Create an SSH tunnel to access the web control panel
-
-   For Carenza environments:
-
-   ```bash
-   $ ssh rabbitmq-1.backend.staging -L 15672:127.0.0.1:15672
-   ```
-
-   For AWS environments:
+Run `gds govuk connect rabbitmq -e integration aws/rabbitmq` and point your
+browser at <http://127.0.0.1:15672>.
 
-   ```bash
-   $ ssh $(ssh integration "govuk_node_list --single-node -c rabbitmq").integration -CNL 15672:127.0.0.1:15672
-   ```
-
-2. Log in to the web control panel
-
-   Point your browser at <http://127.0.0.1:15672>
-
-   The username is root. The password you can obtain from the govuk-secrets
-   repo if you have access. Look for govuk\_rabbitmq::root\_password in the file for the
-   relevant environment in
-   <https://github.com/alphagov/govuk-secrets/tree/master/puppet/hieradata> or <https://github.com/alphagov/govuk-secrets/tree/master/puppet_aws/hieradata>
-
-3. Do your business
-4. Tidy up
-
-   Close the SSH connection you set up earlier with CTRL+C or by typing "exit".
-
-## Inspecting/removing items from a queue
-
-We had an instance where an application was unable to process a message
-in the queue, but left the message on the queue. This meant it was
-backing up. Removing the message from the queue was the right solution
-in our case.
-
-1. Find the queue
-
-   Click on the "Queues" tab. Then click on the name of the queue.
-
-2. Find the messages
-
-   Scroll down and click "Get messages". Clicking the "Get Message(s)"
-   button that appears will fetch however many messages you ask for.
-
-   > **Note**
-   >
-   > Fetching messages actually removes them from the queue. By leaving
-   > the "Requeue" option set to "Yes", they will be added back to queue.
-
-3. Delete the messages
-
-   > **Note**
-   >
-   > There is a risk that you might delete the wrong message(s). This
-   > is because the contents of the queue may have changed.
-
-   Repeat, but change the "Requeue" option to "No".
-
-## Previewing a message for a document_type
```

Comment:
Could we preserve this as part of [the Publishing API doc for RabbitMQ](https://github.com/alphagov/publishing-api/blob/master/docs/rabbitmq.md)?

Reply:
Sure, have added it the the Publishing API doc here -> https://github.com/alphagov/publishing-api/pull/1847